homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Aurora ABB PowerOne Solar Photvoltaic (PV) inverter."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_ADDRESS, CONF_NAME, DEVICE_CLASS_POWER, POWER_WATT)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_COMPORT = 'rs485'
+DEFAULT_ADDRESS = 2
+TEST_MODE = False                      # For testing code when no sunlight.
```

Comment:
Remove debug/test things in the productive env. That is fine for local test but should not go into official PR

Reply:
Done